apitester/apitester/urls.py

```diff
@@ -7,8 +7,37 @@
 
 from base.views import HomeView
 
+from obp.views import OAuthInitiateView, OAuthAuthorizeView, DirectLoginView, GatewayLoginView, LogoutView
+
+from django.conf import settings
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+def LogAtStart():
+    LOGGER.log(logging.INFO, 'OAUTH_BASE_URL: {}'.format(
+        settings.OAUTH_BASE_URL))
+
+    LOGGER.log(logging.INFO, 'API_HOST: {}'.format(
+        settings.API_HOST))
+
+    LOGGER.log(logging.INFO, 'API_ROOT: {}'.format(
+        settings.API_ROOT))
+
+LogAtStart()
+
 urlpatterns = [
     url(r'^$', HomeView.as_view(), name="home"),
-    url(r'^obp/', include('obp.urls')),
```

Comment:
I would advise caution on this for the following reasons:

- It will break existing apitester deployments which expect the `/obp` route to exist  
- Since it's inception the apitester has used the route '/obp' for the return url, any other developers (and including documentation being made) will expect this to still be the case
- It's not clear what bug this commit is fixing (I assume it's related to oauth debugging internally) but the effect of this commit is probably unintentional. 
- I propose there should be consistency on which route oauth apps, be recommending which route apps should delegate for their oauth returl url. 

@PengfeiLi0218 @simonredfern

Reply:
Yeah we'd have to update API Tester configurations on all sandboxes once this gets deployed. Tedious, but IMHO increasing consistency, as none of our other apps need additions to the redirect URL.